Assets/Scripts/Interaction Scripts/ControllerInteractions.cs

```diff
@@ -532,7 +535,8 @@ public void UndoAndDeleteWaypoints()
                     // Otherwise we default to removing the last waypoint (UNDO)
                     Debug.Log("Removing most recently placed waypoint");
 
-                    Waypoint lastWaypoint = (Waypoint)currentlySelectedDrone.waypointsOrder[currentlySelectedDrone.waypointsOrder.Count - 1];
+                    //Waypoint lastWaypoint = (Waypoint)currentlySelectedDrone.waypointsOrder[currentlySelectedDrone.waypointsOrder.Count - 1];
+                    Waypoint lastWaypoint = null;
```

Comment:
why?

Reply:
@apollovision does this not break the undo function?

Reply:
Controller interactions will be depreciated